### ISSUE TITLE
Add data by Baxter 1992

### ISSUE THREAD

**Issue by LinguList** [digling/rhymes#1]
I suppose we take this as the point where we store ALL data on rhymes, and so I should add the Baxter-data which I annotated, also to make it available for the critical inspection by people.

**Comment by LinguList** [comment]
- [ ] provide data in spreadsheet, so that only the readings need to be added (MCH and OCH)

**Comment by LinguList** [comment]
As a next step, Baxter's Old Chinese reconstructions should also be digitized.

**Comment by nh36** [comment]
Hi Marris, 

I think that this dataset already exists, since you used it in your paper, but I don't find it here-- 

https://github.com/digling/rhymes/tree/master/datasets/Baxter1992

which is where I expect it. 

thanks,

**Comment by LinguList** [comment]
Yes, I was lazy in uploading. And we'll also need to add the readings of Baxter in original, which we have not digitized by now.
